templates/zerver/integrations/assembla.md

```diff
@@ -0,0 +1,24 @@
+1. Follow the **Hubot Installation** section of the
```

Comment:
It would be good to add a sentence or two of text, similar to the other integrations in Version Control category.

Reply:
@rishig Are you talking about the ```create-stream.md``` and ```create-bot-construct-url.md```?